pkg/types/powervs/validation/platform.go

```diff
@@ -10,9 +11,24 @@ import (
 func ValidatePlatform(p *powervs.Platform, fldPath *field.Path) field.ErrorList {
 	allErrs := field.ErrorList{}
 
+	//Validate Region
 	if p.Region == "" {
 		allErrs = append(allErrs, field.Required(fldPath.Child("region"), "region must be specified"))
+	} else if _, ok := powervs.Regions[p.Region]; !ok {
+		allErrs = append(allErrs, field.NotSupported(fldPath.Child("region"), p.Region, powervs.RegionShortNames()))
 	}
 
+	//validate ServiceInstanceID
+	if p.ServiceInstanceID != "" {
+		_, err := uuid.Parse(p.ServiceInstanceID)
+		if err != nil {
+			allErrs = append(allErrs, field.Invalid(fldPath.Child("SericeInstanceID"), p.ServiceInstanceID, "ServiceInstanceID must be a valid UUID"))
```

Comment:
typo: "ServiceInstanceID"

Reply:
Fixed

Reply:
gofmt and golint are failing...could you also take a look at those?

Reply:
also the install config tests are failing as this change conflicts with: https://github.com/openshift/installer/blob/master/pkg/types/validation/installconfig_test.go#L92

Reply:
Fixed